ffi/examples/visit-expression/CMakeLists.txt

```diff
@@ -8,26 +8,16 @@ target_link_directories(visit_expression PUBLIC "${CMAKE_CURRENT_SOURCE_DIR}/../
 target_link_libraries(visit_expression PUBLIC delta_kernel_ffi)
 target_compile_options(visit_expression PUBLIC)
 
-target_compile_options(visit_expression PRIVATE -Wall -Wextra -Wpedantic -Werror -Wno-strict-prototypes)
-
-# Get info on the OS and platform
-if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-    set(MACOSX TRUE)
-endif()
-if(UNIX AND NOT APPLE)
-    set(LINUX TRUE)
+if(MSVC)
+  target_compile_options(read_table PRIVATE /W4 /WX)
+else()
+  # no-strict-prototypes because arrow headers have fn defs without prototypes
+  target_compile_options(visit_expression PRIVATE -Wall -Wextra -Wpedantic -Werror -Wno-strict-prototypes -g -fsanitize=address)
+  target_link_options(visit_expression PRIVATE -g -fsanitize=address)
 endif()
 
 # Add the kernel expresion -> engine expression test
 include(CTest)
 set(ExprTestRunner "../../../tests/test-expression-visitor/run_test.sh")
 set(ExprExpectedPath "../../../tests/test-expression-visitor/expected.txt")
 add_test(NAME test_expression_visitor COMMAND ${ExprTestRunner} ${ExprExpectedPath})
-if(LINUX)
-  add_test(NAME test_expression_visitor_leaks COMMAND valgrind
-    --error-exitcode=1
-    --tool=memcheck
-    --leak-check=full
-    --errors-for-leak-kinds=definite
-    --show-leak-kinds=definite ./visit_expression)
-endif()
```

Comment:
@OussamaSaoudi-db I'm not sure if we need to do any kind of cleanup on github actions side, now that this test is removed and we no longer need valgrind?

Reply:
No cleanup necessary 👍